record-and-playback/core/lib/recordandplayback/generators/audio.rb

```diff
@@ -31,43 +31,76 @@ class AudioEvents
     def self.create_audio_edl(events, archive_dir)
       audio_edl = []
       audio_dir = "#{archive_dir}/audio"
+      audios = {}
+      active_audios = []
 
       initial_timestamp = BigBlueButton::Events.first_event_timestamp(events)
       final_timestamp = BigBlueButton::Events.last_event_timestamp(events)
 
       # Initially start with silence
       audio_edl << {
         :timestamp => 0,
-        :audio => nil
+        :audios => nil

```

Comment:
I'd recommend using an empty array `[]` to represent no audios instead of using the value `nil`. (repeated in several places in this file)